Basic Parcels (#17)
agrc/metadata-asset-tracking-tool: metadata/cadastre/utah-parcels.md

# Title

Utah Basic Parcels

## ID

cbe4da2a-3edb-4a4a-b082-60f9371c8006

## Brief Summary

> Parcel boundaries for all 29 counties in Utah that include basic information such as parcel address and generalized land ownership.

Reviewer: ```suggestion
Parcel boundaries for all 29 counties in Utah that include basic information such as parcel ID, parcel address, and generalized land ownership.
```

## Summary

> Utah Basic Parcels contains information such as the county name, parcel address, land owner type, and phone number for the county recorder. Landowners of privately owned parcels are not specified.

Reviewer: ```suggestion
Utah Basic Parcels contain information such as the county FIPS code, parcel ID, parcel address, generalized land ownership type, currency dates, phone number for the county recorder, and county parcel website if available. Landowners of privately owned parcels are not specified.
```

## Description

### What is the dataset?

A parcel is a defined area of land that is owned by a public or private entity. Utah Basic Parcels includes a polygon layer of parcels for each of Utah's 29 counties. Each county provides its individual parcel data to UGRC; thus, users should expect some variation in schema and completeness between counties.

### What is the purpose of the dataset?

These data represents UGRC's ongoing efforts to compile statewide parcels following the passage of [HB113](https://le.utah.gov/~2005/bills/hbillenr/HB0113.htm) in 2005. Parcel data is used in a variety of contexts, including public safety, economic development, transportation, planning, and the provision of public services.

### What does the dataset represent?

> Parcels are represented as polygons. Parcel polygons contain attributes unique to each parcel such as a unique parcel number (PARCEL_ID), address for the parcel (PARCEL_ADD, PARCEL_CITY, PARCEL_ZIP), update information (ParcelsCur, ParcelsRec, ParcelsPub, ParcelYear, ParcelNotes), a generalized land ownership designation (OWN_TYPE), and the phone number for the county recorders office (RECORDER). Fields and field definitions may vary from county to county.

Reviewer: ```suggestion
Parcels are represented as polygons. Parcel polygons contain attributes unique to each parcel such as a unique parcel number (PARCEL_ID), address for the parcel (PARCEL_ADD, PARCEL_CITY, PARCEL_ZIP), update information (ParcelsCur, ParcelsRec, ParcelsPub, ParcelYear, ParcelNotes), a generalized land ownership designation (OWN_TYPE), the phone number for the county recorders office (RECORDER), and a link to the county parcel website (CoParcel_URL) if available. Fields and field definitions may vary from county to county.
```

### How was the dataset created?

<!--- When talking about how the dataset was made, the original metadata talks about the "LIR Working Group", which I presume was the primary force behind the LIR parcels, but was that group also responsible for these basic parcels? Is the history of both datasets about the same? -->

> 

Reviewer: This dataset was compiled from digital parcel data submitted by Utah counties to UGRC.

While addresses are provided with the parcel boundaries the authoritative dataset for addresses from UGRC are the [Utah Address Points](https://gis.utah.gov/products/sgid/location/address-points/).

The owner type field OWN_TYPE in the Basic Parcels is a generalized ownership type (Federal, Private, State, Tribal) and is populated with the value of the OWNER field where the parcel's centroid intersects the [Land Ownership](https://gis.utah.gov/products/sgid/cadastre/land-ownership/).

### How reliable and accurate is the dataset?

These multi-purpose datasets are not a substitute for obtaining the most current, legal land records information from county authorities. They do not represent exact legal boundaries of parcels in Utah and should be used for general analysis and visualization purposes only.

## Credits

### Data Source

- UGRC
- Counties

### Host

UGRC

## Restrictions

## License

## Tags

- Land ownership
- Private land
- Public land

> - HB113

Reviewer: ```suggestion

```

Reviewer: remove this tag

Reviewer 2: it says HB113. Is 433 different?

Reviewer: sorry not seeing straight. It is ok to reference HB113 but not HB433 (all government owned property)
